Reply/app/src/main/java/com/materialstudies/reply/ui/nav/NavigationModel.kt

```diff
@@ -27,9 +27,11 @@ import com.materialstudies.reply.ui.home.Mailbox
  */
 object NavigationModel {
 
+    const val INBOX_ID = 0
```

Comment:
Is there any reason to pull out the other mailbox ids, just for consistency?

Reply:
sure! will do that